packages/openneuro-app/src/scripts/dataset/comments/__tests__/coral-embed.spec.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { render, waitFor } from "@testing-library/react"
+import { CoralEmbed } from "../coral-embed"
+import { UserModalOpenCtx } from "../../../utils/user-login-modal-ctx"
+import { vi } from "vitest"
+
+vi.mock("../../../utils/user-login-modal-ctx", () => ({
+  UserModalOpenCtx: React.createContext({ setUserModalOpen: vi.fn() }),
+}))
+
+global.fetch = vi.fn().mockResolvedValue(Promise.resolve({
+  json: vi.fn().mockResolvedValue({ token: "mock-sso-token" }),
+}))
```

Comment:
This should probably use the fetch mocking library. `vitest-fetch-mock`

Reply:
I am having an issue with vitest-fetch-mock. Is this some sort of version related vite issue?

just adding `import createFetchMock from 'vitest-fetch-mock'; `to my test file I get 

` FAIL  packages/openneuro-app/src/scripts/dataset/comments/__tests__/coral-embed.spec.jsx > CoralEmbed - Simple Embed.js Check (Vitest) > should check if window.Coral exists and initializeCoralEmbed is called
TypeError: Cannot set property testPath of #<Object> which has only a getter`